CHANGELOG.md

```diff
@@ -4,11 +4,17 @@ All notable changes to this project will be documented in this file.
 
 The changelog format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
-## [0.18.1] - 19-Feb-2020
+## [0.18.1] - 24-Feb-2020
 
 :warning: 0.18.x breaks compatibility with the ``profiles`` (public key derivation) saved using previous versions of the software.
 Private keys have now a new public key and address associated.
-Before installing ``nem2-cli@0.18.1``, backup and delete the file  ``~ \.nem2rc.json``.
+Before installing ``symbol-cli@0.18.1``, backup and delete the file  ``~ \.nem2rc.json``.
```

Comment:
should we change the file name to `.symbolrc.json`?

Reply:
The filename has been changed to ``.symbolrc.json``, but they should still remove the file named ``nem2rc.json``.

Reply:
got you!